config/wpcalypso.json

```diff
@@ -114,6 +114,7 @@
 		"onboarding/import-redirect-to-themes": true,
 		"p2/p2-plus": true,
 		"pattern-assembler/add-pages": true,
+		"pattern-assembler/v2": true,
```

Comment:
[nit] It would be better to align with the feature flag `pattern-assembler/v2` and only https://github.com/Automattic/wp-calypso/pull/84960. WDYT?

Reply:
I will disable the flag in Calypso before merging. I like having it enabled here because it helps when testing this PR because we don't need to pass the flag in the URL.